procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,88 @@
+# Council Synchronous Meetings
+
+This documents procedures, norms, and logistics of the Council's sync meetings.
+
+## Frequency
+
+The Council meets once a week.
+
+After the 14th of July, 2023, the Council will re-evaluate whether this frequency still makes sense or whether it should be changed.
```

Comment:
I'd prefer to have an upper bound on this date as well. Could we say "on the first council meeting on or after July 14, 2023, we will re-evaluate this frequency?"